addons/website/static/tests/tours/snippet_table_of_content.js

```diff
@@ -93,9 +77,10 @@ registerWebsitePreviewTour('snippet_table_of_content', {
             }
         },
     },
+    ...clickOnEditAndWaitEditMode(),
```

Comment:
You have to Edit to check mobile mode because we do not have systray item for mobile preview available in non edit mode, we should add systray item instead of editing the view and check mobile mode.

Reply:
@msh-odoo  Yes in either way this functionality can be tested as our main objective is to test  "Check that we have the good TOC on mobile", there shouldn't be any issues.

Reply:
The goal here is to adapt tour, if something is missing then we should add, systray item is missing so we need to implement it if noone is working on it.

Reply:
Okay, in that case i will remove this change and when systray is fixed this tour will autmatically work.